build-system/pr-check/module-build.js

```diff
@@ -53,7 +53,14 @@ function main() {
   } else {
     printChangeSummary(FILENAME);
     const buildTargets = determineBuildTargets(FILENAME);
-    if (buildTargets.has('RUNTIME') || buildTargets.has('FLAG_CONFIG')) {
+    // NOTE: This list must eventually match the same buildTargets check
```

Comment:
Is there a tracking issue for this migration we could TODO here?

Reply:
done. added a TODO instead and a link to the parent PR